src/alertsmanagement/azext_alertsmanagement/_params.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from knack.arguments import CLIArgumentType
+
+
+def load_arguments(self, _):
+
+    name_arg_type = CLIArgumentType(options_list=['--name', '-n'], metavar='NAME')
+
+    action_rule_name = CLIArgumentType(overrides=name_arg_type, help='Name of action rule.',
+                                       id_part='name')
+
+    with self.argument_context('alertsmanagement action-rule create') as c:
+        c.argument('resource_group', resource_group_name_type)
+        c.argument('name', action_rule_name)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+        c.argument('tags', tags_type)
+        c.argument('status', arg_type=get_enum_type(['Enabled', 'Disabled']), id_part=None, help='Indicates if the given action rule is enabled or disabled')
```

Comment:
is there default?

Reply:
You can use `ActionRuleStatus` from src/alertsmanagement/azext_alertsmanagement/vendored_sdks/alertsmanagement/models/_alerts_management_client_enums.py instead of listing the values yourself.

Reply:
> 
> 
> is there default?

Test result shows default value is enabled. Fixed

Reply:
> 
> 
> You can use `ActionRuleStatus` from src/alertsmanagement/azext_alertsmanagement/vendored_sdks/alertsmanagement/models/_alerts_management_client_enums.py instead of listing the values yourself.

Done